sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/scenarios/inc/event_sender.hpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "opentelemetry_helpers.hpp"
+
+#include <azure/core/context.hpp>
+#include <azure/messaging/eventhubs/models/event_data.hpp>
+#include <azure/messaging/eventhubs/models/management_models.hpp>
+#include <azure/messaging/eventhubs/models/partition_client_models.hpp>
+#include <azure/messaging/eventhubs/producer_client.hpp>
+
+#include <memory>
+
+struct EventSenderOptions
+{
+  std::string PartitionId;
+  std::uint32_t MessageLimit;
+  std::uint32_t NumberOfExtraBytes;
+};
+
+class EventSender {
+public:
+  static std::pair<
+      Azure::Messaging::EventHubs::Models::StartPosition,
+      Azure::Messaging::EventHubs::Models::EventHubPartitionProperties>
+  SendEventsToPartition(
+      std::unique_ptr<Azure::Messaging::EventHubs::ProducerClient> const& producerClient,
+      EventSenderOptions const& senderOptions,
+      Azure::Core::Context const& context)
+  {
+    auto sendEventsScope{CreateStressSpan("SendEventsToPartition")};
+
+    std::cout << "[BEGIN] Sending " << senderOptions.MessageLimit << " messages to partition "
+              << senderOptions.PartitionId << ", with messages of size "
+              << senderOptions.NumberOfExtraBytes << std::endl;
```

Comment:
Should this be in the trace as well?

Reply:
It's in the console logs if it matters. Otherwise the information is already in the traces, this would be duplicative.